nanny.toml

```diff
@@ -35,6 +35,14 @@ to=""
 enabled=false
 webhookURL=""
 
+[webhook]
+enabled=false
+webhook_url="https://webhook.somewhere.com/webhook"
+webhook_url_all_clear="https://webhook.somewhere.com/webhook"
+webhook_secret=""
+request_timeout=10
```

Comment:
You can use "10s" and have this be `time.Duration` so that you know what is the unit of this.

Reply:
I have now included the unit. 🙂